webview_panels/src/lib/altimate/main.js

```diff
@@ -6284,7 +6284,7 @@ const ga = { vs: fa, solarizedDark: la, solarizedLight: da },
       },
     }),
   Oa = (e) => rt.get(`dbt/dbt_docs_share/${e}/conversations`),
-  Aa = (e) => rt.get("/users/chat", { company: e }),
+  Aa = (e) => rt.get("users/chat", { company: e }),
```

Comment:
Why do we keep switching?

Reply:
Let's add this.  The api being called right now is `api.myaltimate.com//users/chat` This works ok in prod but local breaks

Reply:
And technically incorrect as well

Reply:
Please also change upstream as this looks like a compiled file.